tests/spread/plugins/crystal/snaps/crystal-hello/hello/src/hello.cr

```diff
@@ -0,0 +1 @@
+puts "hello world"
```

Comment:
Could we use a slightly more complex test with a `require`, just to make sure modules are correctly loaded?

Reply:
The prelude of the std-lib is implicitly required. Otherwise the `puts` usage will not even compile.

Using an external dependency will probably will not be wanted since it will require some external connections, but I could extend the example if wanted. Requiring the sqlite3 connection could be an example (that will also require some addition build_package)

Another alternative is to require something from the std-lib that is outside the prelude (like `Complex`) but that will add no value IMO. Since the prelude is found.

The other thing that can be required is relative files to the project. But I don't think this is something that could break.

Let me know what is preferred.

Reply:
Thanks, if just using `puts` already exercises this, I think it's enough.